tests/meta/workflow/mutant/conftest.py

```diff
@@ -0,0 +1,300 @@
+import pytest
+
+from pathlib import Path
+
+from cg.meta.workflow.mutant.quality_controller.models import (
+    CaseQualityResult,
+    QualityMetrics,
+    SampleQualityResult,
+)
+from cg.meta.workflow.mutant.quality_controller.quality_controller import QualityController
+from cg.meta.workflow.mutant.quality_controller.utils import get_quality_metrics, get_report_path
+from cg.store.models import Case, Sample
+from cg.store.store import Store
+from tests.store_helpers import StoreHelpers
+from cg.constants.constants import ControlOptions
+from tests.mocks.limsmock import LimsSample, LimsUDF, MockLimsAPI
+
+
+@pytest.fixture(name="mutant_analysis_dir_case_qc_pass")
+def mutant_analysis_dir_case_qc_pass(mutant_analysis_dir: Path) -> Path:
```

Comment:
```suggestion
@pytest.fixture
def mutant_analysis_dir_case_qc_pass(mutant_analysis_dir: Path) -> Path:
```

Reply:
None of the fixtures need a name as parameter unless you want to override the method name.

Reply:
Corrected!